### PR TITLE
 CSI Token Passing Provider Support Implemented

### PR DESCRIPTION
**Pull request by Khooshrin**

### CODE REVIEW

File: charts/secrets-store-csi-driver-provider-gcp/Chart.yaml
Comment (amitmodak) [inline charts/secrets-store-csi-driver-provider-gcp/Chart.yaml:1]: This file need not be present in the PR.

File: charts/secrets-store-csi-driver-provider-gcp/values.yaml
Comment (amitmodak) [inline charts/secrets-store-csi-driver-provider-gcp/values.yaml:1]: This file need not be present in the PR.

File: charts/secrets-store-csi-driver-provider-gcp/Chart.yaml
Comment (Khooshrin) [reply charts/secrets-store-csi-driver-provider-gcp/Chart.yaml:1]: Removed.

**Review by amitmodak — APPROVED** [review]
Approved for merging into feature branch to validate tests.

**Pull request merged**